Node/src/botbuilder-teams.d.ts

```diff
@@ -769,6 +769,12 @@ export interface IComposeExtensionResponse {
   composeExtension?: ComposeExtensionResult;
 }
 
+/** Represents the request to a compose extension action command invoke message */
```

Comment:
I don't quite understand what this means. Should the documentation be similar to what ITaskModuleInvokeRequest has? ITaskModuleInvokeRequest: /** Represents the value of the invoke activity of task module fetch and submit. */

Reply:
make sense, fixed